File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0 :
            return False
        num = x
        rev_num = 0
        while x > 0:
            last_dig = x % 10
            rev_num = rev_num * 10 + last_dig
            x = x / 10
        return rev_num == num

        

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
       if(x < 0) return false;
       int num = x;
       int revNum = 0;
       while(x > 0){
        int lastDig = x % 10;
        revNum = revNum * 10 + lastDig;
        x = x / 10;
       } 
       return revNum == num;
    }
}